src/support/bits.cpp

```diff
@@ -54,8 +56,8 @@ template<> int PopCount<uint32_t>(uint32_t v) {
 }
 
 template<> int PopCount<uint64_t>(uint64_t v) {
-#if __has_builtin(__builtin_popcountll) || defined(__GNUC__)
-  return __builtin_popcountll(v);
+#if __has_builtin(__builtin_popcount) || defined(__GNUC__) || defined(_MSC_VER)
```

Comment:
use `__has_builtin(__builtin_popcount)` instead `__has_builtin(__builtin_popcountll)` due to clang-format forcing line terminator and carry `defined(_MSC_VER)` to new line and this looks weird.